index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<!-- <link rel="stylesheet" href="styles.css" /> -->
+		<script type="text/javascript" src="script.js"></script>
+		<title>web-calculator</title>
+		
+	</head>
+	<body>
+		
+		<style>
+		body { background-color: plum; }
```

Comment:
body { background-color: plum; } should be indented a bit further inside of <style></style>